### DISCUSSION TITLE
Max-width control inside widget advanced settings.

### DISCUSSION THREAD

**Discussion by hein-obox** [elementor/elementor#25311] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

For responsive design, I often like to give my widgets a max-width value instead of width value.
The advantage of a max-width value, is that it often makes the element responsive by default.
E.g. a max-width value of 1000px for desktop, works out of the box for smaller device, where it will work as full width, without the necessity of setting a different value for smaller breakpoints.

### Describe the solution you'd like.

See above.

### Describe alternatives you've considered.

Using the width control and setting separate values for large and small breakpoints.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by karolisdailidonis** [comment]
I don't understand why a simple property is not included from the start, like Min Height...?
Or that there is min height but no fixed height..?

**Comment by hein-obox** [comment]
@karolisdailidonis All of this will be solved in the editor V4.
